src/sqlfluff/rules/L057.py

```diff
@@ -89,6 +89,22 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
                     identifier = identifier[:-1]
                 identifier = identifier.replace(".", "")
 
+            # Spark3 file references for direct file query
+            # are quoted in back ticks to allow for identfiers common
+            # in file paths and regex patterns for path globbing
+            # https://spark.apache.org/docs/latest/sql-ref-syntax-qry-select-file.html
+            #
+            # Path Glob Filters (done inline for SQL direct file query)
+            # https://spark.apache.org/docs/latest/sql-data-sources-generic-options.html#path-global-filter  # noqa: E501
```

Comment:
I don't think you need the noqa as long as the URL is the ONLY thing on that line. It's cleaver enough to know that URLs can't be split.

```suggestion
            # https://spark.apache.org/docs/latest/sql-data-sources-generic-options.html#path-global-filter
```